components/card.js

```diff
@@ -27,4 +27,23 @@ export const CardContent = styled.div`
   padding: 40px;
 `;
 
-export default ({ children }) => <Card>{children}</Card>;
+export const CardDiv = styled.div`
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-around;
+    padding: 26px 28px;
+    border-bottom: 2px solid #EDEDED;
+`
+export const TableDiv = styled.div`
+    border: 1px solid #000000;
+    border-radius: 3px;
+    padding: 1px 1px;
+    background-color: #FFFFFF;
+`
+
+
+export default ({children}) => (
+    <Card>
+        {children}
+    </Card>
+)
```

Comment:
are these card changes compatible with everyone else's use of the card component? 

Also card component changes should be in another PR